cumulus_library/actions/uploader.py

```diff
@@ -79,13 +80,15 @@ def upload_files(args: dict):
     if not args["user"] or not args["id"]:
         sys.exit("user/id not provided, please pass --user and --id")
     try:
-        meta_version = next(filter(lambda x: str(x).endswith("__meta_version.parquet"), file_paths))
+        meta_version = next(
+            filter(lambda x: str(x).endswith("__meta_version.meta.parquet"), file_paths)
+        )
```

Comment:
Do we want to look for both filenames? To continue to support the old style too?

Reply:
I don't think so - this was part of the v4 breaking change, any dangling issues related to this should be considered bugs.